hnn_core/network.py

```diff
@@ -412,6 +415,43 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
+    def __eq__(self, other):
```

Comment:
add a test for this method

Reply:
In `net.copy()` method, the simulation data related to `external drives` and `rec_arrays` eg- `events` is deleted.
Is a similar behavior intended for read\write functionality?